calculator.py

```diff
@@ -1,3 +1,15 @@
 def calculator_add(left_hand_number, right_hand_number):
     """ Adds left_hand_number to right_hand_number """
     return left_hand_number + right_hand_number
+def calculator_subtract(left_hand_number, right_hand_number):
```

Comment:
stylistically i'd put one line of vertical space between the end of the last function and beginning of the next.